telemetry/packages/public-app/components/DisplacementChart.tsx

```diff
@@ -0,0 +1,51 @@
+'use client';
+
+import { Card, LineChart, Title } from '@tremor/react';
+import { useQuery } from 'react-query';
+import format from 'date-fns/format';
+
+const valueFormatter = (number: any) =>
+  `$ ${new Intl.NumberFormat('uk').format(number).toString()}`;
+
+export const DisplacementChart = () => {
+  const { data, isLoading, isError } = useQuery(
+    'displacement',
+    async () =>
+      (await fetch(
+        `${process.env.NEXT_PUBLIC_TELEMETRY_SERVER}/pods/pod_1/public-data/displacement?start=0`,
+      ).then((res) => res.json())) as {
+        id: 'displacement';
+        timestamp: string;
+        value: number;
+      }[],
```

Comment:
convert this into a type that is shared with the server for end-to-end type safety (ish)